src/controller/CHIPDeviceController.h

```diff
@@ -712,6 +712,7 @@ class DLL_EXPORT DeviceCommissioner : public DeviceController,
 
     void HandleAttestationResult(CHIP_ERROR err);
 
+public:
```

Comment:
Why?  There's a public API for this (`GetDeviceBeingCommissioned`), which incidentally makes you think about the failure case @cecille raised because it returns an error in that case.  There's a reason these methods are not public.

Reply:
Using the public API now.